packages/bbot/data_stream/asm_intel/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,114 @@
+---
+description: Pipeline for ingesting BBOT ndjson files.
+
+processors:
+####################
+# Event ECS Fields #
+####################
+- set:
+    field: ecs.version
+    value: 8.12.0
+- set:
+    field: event.kind
+    value: asset
+- json:
+    field: message
+    target_field: bbot
+    tag: json_message
+- date:
+    field: bbot.timestamp
+    formats:
+    -   UNIX
+    if: ctx?.bbot?.timestamp != null
```

```suggestion
    if: ctx.bbot?.timestamp != null
```
(similar throughout)